features/security/src/main/feature/feature.xml

```diff
@@ -21,694 +21,317 @@
+        <bundle>mvn:ddf.security.handler/security-handler-basic/${project.version}</bundle>
 
-    <feature name="security-handler-saml" version="${project.version}"
-             description="SAML Web SSO client">
-        <feature>security-core-api</feature>
-        <feature>security-handler-api</feature>
-        <feature>javax-validation</feature>
-        <feature>action-core-impl</feature>
-        <feature>cxf-rs-security-sso-saml</feature>
-        <feature>security-filter-api</feature>
-        <feature>session-invalidator</feature>
-        <feature>platform-paxweb-jettyconfig</feature>
-        <feature>security-filter-login</feature>
-        <feature>guava</feature>
-        <feature>cxf-frontend-javascript</feature>
-        <feature>cxf-jaxrs</feature>
-        <feature>util-uuidgenerator</feature>
-        <feature>security-handler-basic</feature>
-        <feature dependency="true">oidc-dependencies</feature>
-        <feature>security-core-impl</feature>
-        <feature>saml-validator</feature>
-        <feature>security-core-services</feature>
-        <bundle>mvn:ddf.security/security-saml-util/${project.version}</bundle>
-        <bundle>mvn:ddf.security.handler/security-handler-saml/${project.version}</bundle>
-        <bundle>mvn:ddf.security.encryption/security-encryption-api/${project.version}</bundle>
-        <bundle>mvn:ddf.security.expansion/security-expansion-api/${project.version}</bundle>
-        <bundle>mvn:com.google.code.gson/gson/${gson.version}</bundle>
-    </feature>
+        <!-- PKI Auth -->
+        <bundle>mvn:ddf.security.realm/security-realm-pki/${project.version}</bundle>
+        <bundle>mvn:ddf.security.handler/security-handler-pki/${project.version}</bundle>
 
-    <feature name="saml-validator" version="${project.version}" description="Saml token validator">
-        <!--  TODO (https://github.com/codice/ddf/issues/5327)
-                Address potential circular dependencies as part of security refactor  -->
-        <feature>security-core-impl</feature>
-        <feature>security-handler-impl</feature>
-        <bundle>mvn:ddf.security.saml/saml-assertion-validator/${project.version}</bundle>
-        <bundle>mvn:ddf.security.saml/saml-assertion-validator-impl/${project.version}</bundle>
-    </feature>
+        <!-- Email -->
+        <bundle>mvn:ddf.platform.email/platform-email-impl/${project.version}</bundle>
 
-    <feature name="security-command-listener" version="${project.version}"
-             description="Audits all commands entered on the Karaf command line.">
-        <feature>security-core-api</feature>
-        <feature>security-core-impl</feature>
-        <feature>security-core-services</feature>
+        <!-- Miscellaneous -->
+        <bundle>mvn:ddf.platform/platform-paxweb-jettyconfig/${project.version}</bundle>
+        <bundle>mvn:ddf.platform/platform-http-proxy/${project.version}</bundle>
```

Comment:
Was the http proxy enabled by default before? Imo it might be better to leave it uninstalled out of the box, and let people install it if they need it.

Reply:
Another good catch 👍